src/main/java/com/yeahx4/jamza/champion/Champion.java

```diff
@@ -12,6 +10,7 @@
 public class Champion extends Entity  {
     public final Champions type;
     public String nickname;
+    public String location;
```

Comment:
Indicating player's location information in String is not seemed to be good. Get instance of map with String may cause NullPointerException. So try Enum.